dashboard/legacy/middleware/files_api.rb

```diff
@@ -1019,6 +1019,7 @@ def files_put_file(encrypted_channel_id, filename, body)
   MODERATE_THUMBNAILS_FOR_PROJECT_TYPES = %w(
     applab
     gamelab
+    spritelab
```

Comment:
Eventually, we would like to moderate all custom images uploaded in projects. These are the projects that allow custom uploads.